src/net_processing.cpp

```diff
@@ -3502,10 +3510,12 @@ static bool SendRejectsAndCheckIfBanned(CNode* pnode, CConnman* connman)
     AssertLockHeld(cs_main);
     CNodeState &state = *State(pnode->GetId());
 
-    for (const CBlockReject& reject : state.rejects) {
-        // Pass allowOptimisticSend=true here to allow sending the reject message immediately instead of waiting for the
+    if (g_enable_bip61) {
+        for (const CBlockReject& reject : state.rejects) {
+            // Pass allowOptimisticSend=true here to allow sending the reject message immediately instead of waiting for the
         // network thread to do it. This gives us a chance of actually sending out the reject before we close the connection.
         connman->PushMessage(pnode, CNetMsgMaker(INIT_PROTO_VERSION).Make(NetMsgType::REJECT, (std::string)NetMsgType::BLOCK, reject.chRejectCode, reject.strRejectReason, reject.hashBlock), true);
```

Comment:
should fix indents here

Reply:
https://github.com/dashpay/dash/pull/3414/commits/84077095558f72d3ebaeb29f070c6fa8e5b3f30f